src/main/java/uk/gov/hmcts/reform/pip/model/CourtRoom.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.hmcts.reform.pip.model;
+
+import lombok.Builder;
+import lombok.Data;
+import uk.gov.hmcts.reform.pip.model.classified.fields.ClassifiedString;
+import uk.gov.hmcts.reform.pip.model.enums.Classification;
+
+import java.util.List;
+import javax.persistence.AttributeOverride;
+import javax.persistence.AttributeOverrides;
+import javax.persistence.Column;
+import javax.persistence.Embedded;
+import javax.persistence.Id;
+
+/**
+ * This class displays details about the court room.
+ */
+@Data
+@Builder
+public class CourtRoom {
+
+    @Id
+    private Long courtRoomId;
+
+    private Classification classification;
+
+    @Embedded
+    @AttributeOverrides({
+        @AttributeOverride(name = "value", column = @Column(name = "courtRoomName")),
```

Comment:
youve not used the constants in this file whereas you have in the others

Reply:
good spot :) updated